nixos/modules/services/security/vaultwarden/default.nix

```
@@ -230,6 +237,10 @@ in {
       };
       wantedBy = [ "multi-user.target" ];
     };
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.stateDir}' 0750 ${user} ${group} - -"
```

```suggestion
      "dZ '${cfg.stateDir}' 0750 ${user} ${group} - -"
```

You'd likely need `Z` with `DynamicUser`, as the uid/gid are not stable.

---

The module is not using DynamicUser